changelog/1813.trivial.rst

```diff
@@ -0,0 +1,2 @@
+changed Gyroradius::test_value 6 10 and 12 since it errors due to assuming the nonrelativistic case
+added more tests to check functionality for new relativistic features
```

Comment:
I'm wondering if we can delete this changelog entry, since the changes to tests can be considered part of the `breaking` changelog entry.

Reply:
Yeah that makes sense, I agree.